tools/topology/topology2/sof-ace-tplg/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ DEEPBUFFER_FW_DMA_MS=100"
 "cavs-sdw\;mtl-sdw"
 
 # SDW + HDMI topology with passthrough pipelines
-"cavs-sdw\;mtl-sdw-hdmi\;NUM_HDMIS=4"
+"cavs-sdw\;mtl-sdw-hdmi\;NUM_HDMIS=3"
```

Comment:
this topology name needs to be fixed or this needs to moved to a different develop, it's not used by the kernel.